src/components/Navbar.css

```diff
@@ -1,23 +1,48 @@
+header,
+nav,
+ul {
+  display: flex;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 10px 20px 10px;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fff;
+  align-items: center;
+  padding-inline-start: 40px;
+  border-bottom: 1px solid #e8e8e8;
+}
```

Comment:
- [ ] Great job working on this section! 💯 I think you could do small changes to the style to make it look even better and follow the original design. For example, please keep the title in just one line!

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/215217629-0340f66a-5a57-4638-b248-a5e68d569f07.png) | ![image](https://user-images.githubusercontent.com/77069011/215217746-89c689d6-c193-4d9c-b6a0-e8658d5ad8c8.png) |